Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -968,25 +968,46 @@ public void EvaluatePolicyBlockAction()
         [Fact]
         public void TotalUpdatedFungibleAssets()
         {
+            var privateKeys = Enumerable.Range(0, 3).Select(_ => new PrivateKey()).ToList();
+            var gas = Currency.Uncapped("GAS", 0, null);
+
+            var gasFillActions = privateKeys.Select(pk => new UseGasAction()
+            {
+                GasUsage = 0,
+                Memo = "Refill",
+                MintValue = gas * 100,
+                Receiver = pk.Address,
+            });
+
```

Comment:
Maybe we can make a new test for this case, not a fixed previous test.

Reply:
I thought coverage of previous test was too narrow(cannot cover tests with gas).
IMHO, this test have to cover cases with gas, since `TotalUpdatedFungibleAssets` is needed just for the purpose of gas-refilling, as I know. (thought previous test was useless)